docs/user/intro.rst

```diff
@@ -38,10 +38,11 @@ But there is more.. you can also give your module a dictionary in an interactive
 
 ::
 
-    >>> from mymodule import MyModule
+    >>> from argschema import ArgSchemaParser
+    >>> from mymodule import MySchema
     >>> d = {'a':5}
-    >>> mod = MyModule(input_data=d)
-    >>> mod.run()
+    >>> mod = ArgSchemaParser(input_data=d,schema_type=MySchema)
```

Comment:
Is the argument really a "schema_type" or just a "schema"?  It is a type in that the argument really is a class (that could be instantiated).  On the other hand, the class *is* the "schema".  So it might read nicer ", schema=MySchema".